flake.nix

```diff
@@ -61,6 +62,7 @@
       ) [
         pkgs.nixUnstable
         nixpkgs-nix-unstable-20200829.legacyPackages.${system}.nixUnstable
+        nixpkgs-nix-unstable-20200829.legacyPackages.${system}.nixUnstable
```

Comment:
I think this was meant to be `nixpkgs-nix-unstable-20201205` instead?

Reply:
Nice catch, thanks!